DependencyInjection/CydricknOpenApiValidatorExtension.php

```diff
@@ -58,8 +63,17 @@ protected function loadInternal(array $mergedConfig, ContainerBuilder $container
         ]);
         $container->setDefinition('cydrickn.openapi_validator.validator', $validator);
 
+        if ($conditionInfo = $mergedConfig['condition'] ?? null) {
```

Comment:
Hi @akaffenberger ,

Maybe you can change this to.

```suggestion
        $conditionInfo = $mergedConfig['condition'] ?? null;
        if ($conditionInfo !== null) {
```